source/assets/javascripts/application.js

```diff
@@ -20,3 +20,9 @@ if (waypointElement !== null) {
 if (/iPad|iPhone|iPod/.test(navigator.userAgent)) {
   document.documentElement.className += " ios-device";
 }
+
+jQuery.get("http://ipinfo.io", function(response) {
+  if (response.country == 'ZA') {
+    $("#London").insertAfter("#CapeTown");
+  }
+}, "jsonp");
```

Comment:
This is going to happen on every page load, but we only want it on the contact page.  Can we do something to make it only trigger on that page?  Or only on pages that have the contact details in them?  Look at the code above for waypoints - it's only going to trigger when there's an element with a data-header-waypoint attribute on the page, we could do something similar.

I'm also a wee-bit wary of adding the jquery dependency for just this one thing - admittedly it's giving us the ajax request + the dom manipulation stuff which are super convenient and easier than coding up alternatives ourselves, but maybe there's a smaller dependency we can add instead?  Does @tomsabin have any ideas?